web/flat-web/src/pages/BigClassPage/index.tsx

```diff
@@ -294,8 +294,11 @@ export const BigClassPage = observer<BigClassPageProps>(function BigClassPage()
                 {whiteboardStore.isWritable && (
                     <TopBarRightBtn
                         title="Vision control"
-                        icon="follow"
-                        active={whiteboardStore.viewMode === ViewMode.Broadcaster}
+                        icon={
+                            whiteboardStore.viewMode === ViewMode.Broadcaster
```

Comment:
remove vision control in the Multi-Window

Reply:
this is main branch, not is dev branch